### DISCUSSION TITLE
Can I use the Sirius Web-based project as an extension for VS Code or Theia?

### DISCUSSION THREAD

**Discussion by lak-proddev** [eclipse-sirius/sirius-components#3111] [Q&A]
Can I use the Sirius Web-based Graphical editor as an extension for VS Code or Theia?

**Comment by pcdavid** [comment]
There is a VS Code extension version (of the client part) in https://github.com/eclipse-sirius/sirius-web/tree/master/vscode-extension. It should work with Theia but we have not tested that recently.

There is a version available in the VS Code marketplace at https://marketplace.visualstudio.com/items?itemName=eclipse-sirius.sirius-web, but it is rather old.

To test a more recent version is you will need to rebuild it yourself.

Note that the extension only includes the client/frontend part, to use it you will still need to have a Sirius Web server running and connect to it.

**Reply by lak-proddev** [reply]
Thank you very much for the quick reply.
